client/src/modules/Opportunities/components/ViewCv/ActionButtons/index.test.tsx

```diff
@@ -45,10 +45,10 @@ describe('ActionButtons', () => {
 
     fireEvent.click(shareButton);
 
-    const successNotification = screen.getByText('Copied to clipboard');
+    // const successNotification = screen.getByText('Copied to clipboard');
 
     expect(mockCopyToClipboard).toHaveBeenCalledWith(mockUrl);
-    expect(successNotification).toBeInTheDocument();
+    // expect(successNotification).toBeInTheDocument();
```

Comment:
plus `async` at line 41
```suggestion
    const successNotification = await screen.findByText('Copied to clipboard');

    expect(mockCopyToClipboard).toHaveBeenCalledWith(mockUrl);
    expect(successNotification).toBeInTheDocument();
```